e2e/testQA9-109.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { test, expect } from "@playwright/test"
+import { PageManager } from "../page-objects/pageManager"
+import { advancedSearchKeywords } from "../helper/data"
+
+let pm: PageManager
+
+test.beforeEach(async ({ page }) => {
+    pm = new PageManager(page)   // Create a page manager for interactions
+    await page.goto('/')
+    await expect(pm.onHomepage().storeLogo).toBeVisible() // Assert that the site is opened
+})
+
+test('Advanced search for products with expected product name keywords', async ({ page }) => {
```

Comment:
It seems like the "page" parameter in your async function is unused. You might want to remove it if it's not necessary for the function's operation.